lib/friendly_shipping/services/ups_json.rb

```diff
@@ -0,0 +1,223 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'friendly_shipping/http_client'
+require 'friendly_shipping/services/ups_json/api_error'
+require 'friendly_shipping/services/ups_json/generate_rates_payload'
+require 'friendly_shipping/services/ups_json/parse_json_response'
+require 'friendly_shipping/services/ups_json/parse_money_hash'
+require 'friendly_shipping/services/ups_json/parse_rates_response'
+require 'friendly_shipping/services/ups_json/rates_item_options'
+require 'friendly_shipping/services/ups_json/rates_package_options'
+require 'friendly_shipping/services/ups_json/rates_options'
+require 'friendly_shipping/services/ups_json/shipping_methods'
+
+module FriendlyShipping
+  module Services
+    class UpsJson
+      include Dry::Monads[:result]
+
+      attr_reader :token, :test, :client
+
+      SHIPPING_METHODS = FriendlyShipping::Services::UpsJson::SHIPPING_METHODS
```

Comment:
I think this line can be taken out. `SHIPPING_METHODS` is already defined in this namespace.